plugins/plugin-python/che-plugin-python-lang-server/pom.xml

```diff
@@ -64,5 +60,13 @@
             <groupId>org.eclipse.che.plugin</groupId>
             <artifactId>che-plugin-python-lang-shared</artifactId>
         </dependency>
+		<dependency>
```

Comment:
I think it is sorted: che-plugin...<org.eclipse and org.eclipse.che < org.eclipse.lsp.